collect_app/src/main/res/values/style.xml

```diff
@@ -1,6 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <style name="BaseWidgetStyle">
+        <item name="android:layout_marginLeft">7dp</item>
+        <item name="android:layout_marginTop">5dp</item>
+        <item name="android:layout_marginRight">7dp</item>
+        <item name="android:layout_marginBottom">5dp</item>
+        <item name="android:padding">16dp</item>
```

Comment:
And what about the padding? That button looks a lot taller than other widget buttons to me!